server/startup/prerender.js

```diff
@@ -3,9 +3,12 @@ import { WebApp } from "meteor/webapp";
 import { Logger } from "/server/api";
 
 export default function preRender() {
-  if (process.env.PRERENDER_TOKEN && process.env.PRERENDER_HOST) {
+  if (process.env.PRERENDER_TOKEN && process.env.PRERENDER_SERVICE_URL) {
     prerender.set("prerenderToken", process.env.PRERENDER_TOKEN);
-    prerender.set("host", process.env.PRERENDER_HOST);
+    prerender.set("prerenderServiceUrl", process.env.PRERENDER_SERVICE_URL);
+    if (process.env.PRERENDER_HOST) {
```

Comment:
You are looking for `process.env.PRERENDER_HOST` here, but you are already guarding for only having `process.env.PRERENDER_TOKEN && process.env.PRERENDER_SERVICE_URL` so this code would never get executed?

Reply:
`PRERENDER_HOST` is just an optional parameter. Both `PRERENDER_TOKEN` AND `PRERENDER_SERVICE_URL` are required. I don't think it makes sense to add `PRERENDER_HOST` without these 2 required values.

Reply:
Isn't `PRERENDER_HOST` the old version of `PRERENDER_SERVICE_URL`? Why would we allow both?

Reply:
I dont think so. Looking at the history of https://github.com/prerender/prerender-node, I see it has always been `PRERENDER_SERVICE_URL` to specify the prerender server. `PRERENDER_HOST` is for another purpose which is to specify the site URL.

Reply:
I am confused by why we grouped those two parameters together. They are both required? If so should we throw an error if one is provided and the other is not?